src/app/core/entity/entity-actions/entity-actions.service.ts

```diff
@@ -102,9 +104,22 @@ export class EntityActionsService {
       $localize`:Entity action progress dialog:Processing ...`,
     );
     let result = new CascadingActionResult();
+
     for (let entity of entities) {
+      if ("User" === entity.getType()) {
+        this.keycloakAuthService.deleteUser(entity.getId()).subscribe({
+          next: () => {},
+          error: async () =>
+            await this.confirmationDialog.getConfirmation(
+              $localize`:delete account in keycloak related error title:Keycloak User could not be deleted`,
+              $localize`:delete account in keycloak related error dialog:User Account could not be deleted in Keycloak. Please delete user manually in Keycloak.`,
+              OkButton,
+            ),
+        });
+      }
       result.mergeResults(await this.entityDelete.deleteEntity(entity));
```

Comment:
wouldn't this code logically rather belong in the EntityDeleteService, one layer down?